### PR TITLE
Add upgrade_data_<VER>_<STORAGE> tests execution possibility

### PR DESCRIPTION
**Pull request by KVS85**
http://jenkins01.arangodb.biz:8080/view/3.3/job/arangodb-3.3-linux-matrix/392/
http://jenkins01.arangodb.biz:8080/view/3.3/job/arangodb-3.3-windows-matrix/360/
http://jenkins01.arangodb.biz:8080/view/3.3/job/arangodb-3.3-mac-matrix/248/

### CODE REVIEW

File: js/client/modules/@arangodb/testsuites/upgrade-data.js [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+};
+
+////////////////////////////////////////////////////////////////////////////////
+/// set up the test according to the testcase.
+////////////////////////////////////////////////////////////////////////////////
+const unpackOldData = (engine, version, options, serverOptions) => {
+  const archiveName = `upgrade-data-${version}-${engine}`;
+  const dataFile = fs.join(options.upgradeDataPath,
+    'data',
+    `${archiveName}.tar.gz`);
```
Comment (dothebart) [inline js/client/modules/@arangodb/testsuites/upgrade-data.js]: @Luschgy does this work on the windows nodes?

File: js/client/modules/@arangodb/testing.js [hunk 2]
```diff
@@ -459,8 +459,16 @@ function unitTest (cases, options) {
   // tests to run
   let caselist = [];
 
+  const expandWildcard = ( name ) => {
```
Comment (dothebart) [inline js/client/modules/@arangodb/testing.js]: this is unrelated? But I like it ;-)

**Comment by danielhlarkin** [comment]
Removed the appropriate suites from the Oskar blacklist and added the `upgrade-data-tests` subrepo to the `.gitignore`.

File: js/client/modules/@arangodb/testsuites/upgrade-data.js (hunk 1 above)
Comment (KVS85) [reply js/client/modules/@arangodb/testsuites/upgrade-data.js]: AFAIK, `tar.exe` is preinstalled on Windows nodes.

**Pull request merged**